api/models/comments-models.js

```diff
@@ -1,9 +1,34 @@
 const db = require("../../db/connection");
+const format = require("pg-format");
+
+exports.insertCommentByArticleId = (article_id, { username, body }) => {
+  const sqlStatement = format(
+    `
+  INSERT INTO comments
+    (article_id, author, body)
+  VALUES
+    (%s, %L, %L)
+  RETURNING *;`,
```

Comment:
We are potentially open to injection here with the use of %s which will not escape characters like  a single quote which could change the nature of the sql query. We don't actually need to use pg-format here as node-pg gives us parametrised queries out of the box.

Reply:
I've replaced this with a standard query statement now.  Maybe one for me to ask about in 'After Dark' to get clarification on when to use pg-format